internal/app/siftool/info.go

```diff
@@ -11,26 +11,70 @@ package siftool
 import (
 	"fmt"
 	"io"
-	"time"
+	"strings"
+	"text/tabwriter"
 
 	"github.com/sylabs/sif/v2/pkg/sif"
 )
 
+// readableSize returns the size in human readable format.
+func readableSize(size uint64) string {
+	var divs int
+
+	for ; size != 0; size >>= 10 {
+		if size < 1024 {
+			break
+		}
+		divs++
+	}
+
+	switch divs {
+	case 4:
+		return fmt.Sprintf("%dTB", size)
```

Comment:
With powers of 1024 these should probably be `TiB` etc?

Reply:
Found a couple of other issues with this code too (#54, #55). Addressed those and added unit tests.